src/content/en/showcase/2017/bear71.md

```diff
@@ -0,0 +1,50 @@
+
+## Challenge
+
+The original Bear 71 was developed in Flash; arguably the best interactive storytelling medium of the time. A lot of care went into the technology and it was considered cutting edge. 5 Years later, the original vision holds up and story is still very relevant, but the technology behind it needs an update. Any technology, used creatively, can assist a good story, but Virtual Reality deserves special attention. VR has come and gone several times before, but it’s finally arrived in the mainstream. This is a huge opportunity for storytelling. Stories which traditionally happened in front of you, can now happen around you in VR.
+
+When Bear 71 was built, Adobe Flash was in its golden years. Each passing year, Flash's market share shrinks, and with that, previously accessible works are harder and harder to view. But with Flash's decline has come the maturation of HTML 5. Furthermore, VR is being taken more seriously than ever as a real platform for storytelling.
+
+To our eyes and ears, VR is naturally immersive. But VR has several barriers in its way. At first glance, the variety of headsets should suit many needs and budgets, and the variety of app stores involved in distribution is more choice. But users are known to prefer to do as much in a browser as possible. Users prefer to download and install as little as possible. For users and content creators alike, the appeal of HTML5 is the fact that rich experiences are just one URL away. This is where WebVR comes in.
+
+WebVR enables immersive and comfortable VR content in your browser, across a multitude of software and hardware.
+
+The NFB recognized this opportunity and considered giving Bear 71 a new life in HTML5 and WebVR. This opportunity wouldn't be without a new challenge, common to many VR projects: How do we achieve beautiful things, at good framerates, in stereo vision, on various platforms including mobile phones?
+
+
+## Solution
+
+During the start of development in late 2016, WebVR was not yet readily available. The WebVR API is evolving quickly, and the WebVR standard is currently being drafted. But that didn't stop the community from patching together a suitable stopgap: the WebVR polyfill. It provided useable head tracking and stereo rendering via the gyroscope and WebGL. This polyfill, along with Google's cardboard, allowed us to start building and testing content. When Google Pixel and Daydream became available, along with the first real builds of WebVR, our content was ready to take advantage of it. For the purposes of quick development, we still include and use the polyfill where WebVR is not yet available. For instance, we do much of our developing and debugging on chrome 55’s mobile emulator. Also, it was common to develop and debug on whatever was readily available, including but not limited to HTC One M9, iPhone 5S, Samsung Galaxy and of course Google Pixel. The versatility of the polyfill cannot be understated.
```

Comment:
Brand fixes, should be:
Google Cardboard
When Daydream View became available
first real builds of WebVR in Chrome
Chrome, not chrome
Pixel by Google